connectors/sources/outlook.py

```diff
@@ -735,45 +857,53 @@ def get_default_configuration(cls):
                 "sensitive": True,
                 "type": "str",
             },
+            "client_emails": {
+                "depends_on": [{"field": "data_source", "value": OUTLOOK_CLOUD}],
+                "label": "Client Email Addresses (comma-separated)",
+                "order": 5,
+                "tooltip": "Specify the email addresses to limit data fetching to specific clients. If left empty, data will be fetched for all users.",
+                "required": False,
+                "type": "list",
+            },
```

Comment:
We normally handle lists like this with `*` being default value and `*` meaning that everything should be exported.

See this as an example: https://github.com/elastic/connectors/blob/main/connectors/sources/sharepoint_online.py#L1192-L1199

Reply:
I've applied the suggested changes and updated the logic to the WILDCARD `*` when client_emails is not specified 
920697a